### DISCUSSION TITLE
如何在qiankun 基座中禁用掉子应用 根据router配置的name修改了document.title的能力

### DISCUSSION THREAD

**Discussion by sdmu-gaoqi** [umijs/umi#12174] [General]
在基座中无法获取子应用的router进行 更新 如何在基座中配置禁止子应用运行时根据route 的name动态修改title

**Comment by fz6m** [comment]
使用了 max 自带的 layout 菜单就会自动更改页面名字，你在子应用使用了 layout 的菜单吗？

可以通过 `pageTitleRender` 这个选项自定义显示的标题内容，详见 `ProLayout` [文档](https://procomponents.ant.design/components/layout)。

配置参考：

```ts
// src/app.ts

export const layout = () => {
  return {
    pageTitleRender: (...) => {
      return '你的标题'
    }
  };
};
```
